tlsobs-api/handlers.go

```diff
@@ -636,6 +638,45 @@ func PreflightHandler(w http.ResponseWriter, r *http.Request) {
 	w.Write([]byte("preflighted"))
 }
 
+func heartbeatHandler(w http.ResponseWriter, r *http.Request) {
+	val := r.Context().Value(ctxDBKey)
+	if val == nil {
+		w.WriteHeader(http.StatusInternalServerError)
+		return
+	}
+	db, ok := val.(*pg.DB)
+	if !ok || db.Ping() != nil {
+		w.WriteHeader(http.StatusInternalServerError)
+		return
+	}
+	w.WriteHeader(http.StatusOK)
+}
+
+func versionHandler(w http.ResponseWriter, r *http.Request) {
+	dir, err := os.Getwd()
+	if err != nil {
+		w.WriteHeader(http.StatusInternalServerError)
+		return
+	}
+	filename := path.Clean(dir + string(os.PathSeparator) + "version.json")
```

Comment:
Could you add that file to the repo to prevent the api from crashing when people run it directly?